src/gpu-compute/register_file_cache.cc

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
Copyright should be different here since we wrote it, not AMD.  See this file for an example: https://github.com/gem5/gem5/blob/develop/src/mem/ruby/structures/ALUFreeListArray.cc#L2